src/libtrx/config/map_tr2.def

```diff
@@ -22,6 +22,8 @@ CFG_BOOL(g_Config, visuals.enable_exit_fade_effects, true)
 CFG_BOOL(g_Config, visuals.fix_item_rots, true)
 CFG_INT32(g_Config, visuals.fov, 80)
 CFG_BOOL(g_Config, visuals.use_pcx_fov, true)
+CFG_INT32(g_Config, visuals.fog_start, 12)
```

Comment:
Should we raise the defaults to match TR1?

Reply:
Yeah let's do that.